### PR TITLE
Add ipyslurm

### PR DESCRIPTION
**Pull request by auneri**
Request to add [ipyslurm](https://pypi.org/project/ipyslurm) to conda-forge

### CODE REVIEW

File: recipes/ipyslurm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    - ipython
+    - ipywidgets
+    - notebook
+    - paramiko
+    - python >= 2.7
+    - six
+    - tqdm
+
+test:
+  commands:
```
Comment (igortg) [inline recipes/ipyslurm/meta.yaml]: Instead `commands` you can use:
```
test:
  imports:
    - ipyslurm
```

**Comment by igortg** [comment]
@conda-forge-admin, please rerender

**Comment by igortg** [comment]
CI is not finding any file to build. Try to add the `fn` keyword in the `source` section of `meta.yml`.

**Comment by igortg** [comment]
Build still gives `invalid recipe dir: /home/conda/conda-recipes/ipyslurm - skipping`. I guess this is not right. 

Anyone from @conda-forge/staged-recipes may check?

File: recipes/ipyslurm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - pip
+    - python >= 2.7
```
Comment (isuruf) [inline recipes/ipyslurm/meta.yaml]: should be `python >=2.7`

**Comment by auneri** [comment]
Thank you for reviewing!

**Pull request merged**